ddtrace/settings/config.py

```diff
@@ -517,7 +517,7 @@ def __init__(self):
         self._telemetry_install_type = os.getenv("DD_INSTRUMENTATION_INSTALL_TYPE", None)
         self._telemetry_install_time = os.getenv("DD_INSTRUMENTATION_INSTALL_TIME", None)
 
-    def __getattr__(self, name):
+    def __getattr__(self, name) -> Any:
```

Comment:
🤔  can't this type be made more specific?

Reply:
Probably yes - it has to encapsulate all of the types stored as attributes on the Config object.

Reply:
We could probably restrict it to IntegrationConfig and _JSONType

Reply:
I might address this in a future pull request